examples/all/manifests/thanos-receive-statefulSet.yaml

```diff
@@ -61,6 +61,14 @@ spec:
         - mountPath: /var/thanos/tsdb
           name: data
           readOnly: false
-      volumes:
-      - emptyDir: {}
-        name: data
+      volumes: null
```

Comment:
Is this intentionally null?

Reply:
Yes, I guess, because we set an emptyDir in the version without a PVC. We could maybe filter the volumes key. But instead we simply overwrite to null right now. Worked so far. :man_shrugging: